libraries/rush-lib/src/logic/operations/AsyncOperationQueue.ts

```diff
@@ -177,11 +168,12 @@ export class AsyncOperationQueue
 
   public tryGetRemoteExecutingOperation(): OperationExecutionRecord | undefined {
     const { _queue: queue } = this;
+
     // cycle through the queue to find the next operation that is executed remotely
     for (let i: number = queue.length - 1; i >= 0; i--) {
       const operation: OperationExecutionRecord = queue[i];
 
-      if (operation.status === OperationStatus.RemoteExecuting) {
+      if (operation.status === OperationStatus.RemoteExecutingPossiblyComplete) {
         return operation;
       }
     }
```

Comment:
This entire function is no longer needed, since remote executing operations to be checked are part of the primary loop.